docs/docs/reference/configurations.md

```diff
@@ -97,6 +97,13 @@ Toggle CORS check function. Set `true` to check each request origin. Set `false`
 * Type: `Optional`
 * Default value: `true`
 
+### `SERVER_DEDUP_ENABLED`
+
+The server decides whether or not to handle the duplicate requests for the active connection based on this parameter. Set `false` to disbale the setting.
```

Comment:
... handle duplicate batches for the active connection based on the... Within a connection, if a batch is sent with duplicate **ReqGUID** , the server provides best effort basis to remove the duplicate batches.

Reply:
done.